tests/llm/test_embedding_client.py

```
@@ -12,8 +12,17 @@ def __init__(self, embedding: list[float]) -> None:
         self.embedding: list[float] = embedding
         self.calls: list[dict[str, Any]] = []
 
-    async def create(self, *, model: str, input: str | list[str]) -> SimpleNamespace:
-        self.calls.append({"model": model, "input": input})
+    async def create(
+        self,
+        *,
+        model: str,
+        input: str | list[str],
```

Ruff (0.15.10)

[error] Function argument `input` is shadowing a Python builtin

(A002)